indico/modules/events/papers/templates/_contributions.html

```diff
@@ -47,15 +47,22 @@ <h3 class="icon-file">{% trans %}Paper{% endtrans %}</h3>
                 <h2>{% trans %}Editing{% endtrans %}</h2>
                 </div>
             </div>
-            {% for editable in contribution.editables -%}
-                <div>
-                    <a href="{{ url_for('event_editing.editable', editable) }}">
-                        {%- trans type=editable.type.name %}
-                            Go to timeline ({{ type }})
-                        {% endtrans -%}
-                    </a>
-                </div>
-            {%- endfor %}
+            <div class="action-box">
+                {% for editable in contribution.editables -%}
+                    {% set latest_revision = editable.revisions|last %}
+                    {% set latest_state = latest_revision.final_state or latest_revision.initial_state %}
+                    <div class="section f-j-space-between">
+                        <div class="i-tag {{ latest_state.css_class }}">
+                            {%- trans type=editable.type.title %} {{ type }} {% endtrans -%}: {% trans state=latest_state.title %} {{ state }} {% endtrans -%}
```

Comment:
None of these trans blocks here are needed

Reply:
Why is it needed [here](https://github.com/indico/indico/blob/master/indico/modules/events/papers/templates/_contributions.html#L54)?

Reply:
There you have text that can be translated: `Go to timeline: {type}`

But in this case you only have a variable (which is already a translatable string itself)